common/scripts/pre-commit/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "common-scripts-precommit",
+  "version": "0.0.0",
+  "scripts": {
+    "build": "echo OK",
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
I don't think they're needed. If they are, we might also need the _phase:* commands

Reply:
Both are required - otherwise building the repo errors out with `ERROR: The project 'common-scripts-precommit' does not define a 'build' command in the 'scripts' section of its package.json` (same for test)